### PR TITLE
SUS | get rid of sonar issues

### PR DESCRIPTION
**Pull request by bognix**
@ludwikkazmierczak

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/testcases/chattests/ChatTests.java [hunk 1]
```diff
@@ -14,8 +13,10 @@
 
 public class ChatTests extends NewTestTemplate {
 
-  private String MESSAGE_ON_MAIN_CHAT = "Test message on main chat";
-  private String MESSAGE_ON_PRIVATE_CHAT = "Test message on private chat";
+  private final String USER_IN_PRIVATE_SECTION_NOT_DISPLAYED_ERROR = "USER IS NOT DISPLAYED IN PRIVATE SECTION";
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/chattests/ChatTests.java]: MAJOR Rename this field "USER_IN_PRIVATE_SECTION_NOT_DISPLAYED_ERROR" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS00116)
MAJOR Make this final field static too. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1170)

File: src/test/java/com/wikia/webdriver/testcases/chattests/ChatTests.java [hunk 2]
```diff
@@ -14,8 +13,10 @@
...
+  private final String MESSAGE_ON_MAIN_CHAT = "Test message on main chat";
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/chattests/ChatTests.java]: MAJOR Make this final field static too. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1170)

File: src/test/java/com/wikia/webdriver/testcases/chattests/ChatTests.java [hunk 3]
```diff
@@ -14,8 +13,10 @@
...
+  private final String MESSAGE_ON_PRIVATE_CHAT = "Test message on private chat";
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/chattests/ChatTests.java]: MAJOR Make this final field static too. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1170)

File: src/test/java/com/wikia/webdriver/testcases/chattests/ChatTests.java [hunk 4]
```diff
@@ -14,8 +13,10 @@
...
+  private final String MESSAGE_ON_CHAT_NOT_DISPLAYED_ERROR = "MESSAGE ON CHAT IS NOT DISPLAYED";
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/chattests/ChatTests.java]: MAJOR Make this final field static too. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1170)
MAJOR Rename this field "MESSAGE_ON_CHAT_NOT_DISPLAYED_ERROR" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS00116)

File: src/test/java/com/wikia/webdriver/testcases/chattests/ChatTestsStaff.java [hunk 5]
```diff
@@ -11,8 +11,9 @@
 
 public class ChatTestsStaff extends NewTestTemplate {
 
-  private String MESSAGE_ON_MAIN_CHAT = "Test message on main chat";
-  private String MESSAGE_ON_PRIVATE_CHAT = "Test message on private chat";
+  private final String MESSAGE_ON_MAIN_CHAT = "Test message on main chat";
+  private final String MESSAGE_ON_PRIVATE_CHAT = "Test message on private chat";
+  private final String MESSAGE_ON_CHAT_NOT_DISPLAYED_ERROR = "MESSAGE ON CHAT IS NOT DISPLAYED";
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/chattests/ChatTestsStaff.java]: MAJOR Rename this field "MESSAGE_ON_CHAT_NOT_DISPLAYED_ERROR" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS00116)
MAJOR Make this final field static too. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1170)

File: src/test/java/com/wikia/webdriver/testcases/chattests/ChatTestsStaff.java [hunk 6]
```diff
@@ -11,8 +11,9 @@
 
 public class ChatTestsStaff extends NewTestTemplate {
 
-  private String MESSAGE_ON_MAIN_CHAT = "Test message on main chat";
-  private String MESSAGE_ON_PRIVATE_CHAT = "Test message on private chat";
+  private final String MESSAGE_ON_MAIN_CHAT = "Test message on main chat";
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/chattests/ChatTestsStaff.java]: MAJOR Make this final field static too. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1170)

File: src/test/java/com/wikia/webdriver/testcases/chattests/ChatTestsStaff.java [hunk 7]
```diff
@@ -11,8 +11,9 @@
...
+  private final String MESSAGE_ON_PRIVATE_CHAT = "Test message on private chat";
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/chattests/ChatTestsStaff.java]: MAJOR Make this final field static too. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1170)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 10 issues:
* MAJOR 10 major

Watch the comments in this conversation to review them.

File: src/test/java/com/wikia/webdriver/testcases/chattests/ChatTests.java [hunk 8]
```diff
@@ -14,8 +13,10 @@
 
 public class ChatTests extends NewTestTemplate {
 
-  private String MESSAGE_ON_MAIN_CHAT = "Test message on main chat";
-  private String MESSAGE_ON_PRIVATE_CHAT = "Test message on private chat";
+  private final static String USER_IN_PRIVATE_SECTION_NOT_DISPLAYED_ERROR = "USER IS NOT DISPLAYED IN PRIVATE SECTION";
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/chattests/ChatTests.java]: MINOR Reorder the modifiers to comply with the Java Language Specification. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AModifiersOrderCheck)

File: src/test/java/com/wikia/webdriver/testcases/chattests/ChatTests.java [hunk 9]
```diff
@@ -14,8 +13,10 @@
...
+  private final static String MESSAGE_ON_MAIN_CHAT = "Test message on main chat";
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/chattests/ChatTests.java]: MINOR Reorder the modifiers to comply with the Java Language Specification. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AModifiersOrderCheck)

File: src/test/java/com/wikia/webdriver/testcases/chattests/ChatTests.java [hunk 10]
```diff
@@ -14,8 +13,10 @@
...
+  private final static String MESSAGE_ON_PRIVATE_CHAT = "Test message on private chat";
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/chattests/ChatTests.java]: MINOR Reorder the modifiers to comply with the Java Language Specification. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AModifiersOrderCheck)

File: src/test/java/com/wikia/webdriver/testcases/chattests/ChatTests.java [hunk 11]
```diff
@@ -14,8 +13,10 @@
...
+  private final static String MESSAGE_ON_CHAT_NOT_DISPLAYED_ERROR = "MESSAGE ON CHAT IS NOT DISPLAYED";
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/chattests/ChatTests.java]: MINOR Reorder the modifiers to comply with the Java Language Specification. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AModifiersOrderCheck)

File: src/test/java/com/wikia/webdriver/testcases/chattests/ChatTestsStaff.java [hunk 12]
```diff
@@ -11,8 +11,9 @@
 
 public class ChatTestsStaff extends NewTestTemplate {
 
-  private String MESSAGE_ON_MAIN_CHAT = "Test message on main chat";
-  private String MESSAGE_ON_PRIVATE_CHAT = "Test message on private chat";
+  private final static String MESSAGE_ON_MAIN_CHAT = "Test message on main chat";
+  private final static String MESSAGE_ON_PRIVATE_CHAT = "Test message on private chat";
+  private final static String MESSAGE_ON_CHAT_NOT_DISPLAYED_ERROR = "MESSAGE ON CHAT IS NOT DISPLAYED";
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/chattests/ChatTestsStaff.java]: MINOR Reorder the modifiers to comply with the Java Language Specification. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AModifiersOrderCheck)

File: src/test/java/com/wikia/webdriver/testcases/chattests/ChatTestsStaff.java [hunk 13]
```diff
@@ -11,8 +11,9 @@
 
 public class ChatTestsStaff extends NewTestTemplate {
 
-  private String MESSAGE_ON_MAIN_CHAT = "Test message on main chat";
-  private String MESSAGE_ON_PRIVATE_CHAT = "Test message on private chat";
+  private final static String MESSAGE_ON_MAIN_CHAT = "Test message on main chat";
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/chattests/ChatTestsStaff.java]: MINOR Reorder the modifiers to comply with the Java Language Specification. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AModifiersOrderCheck)

File: src/test/java/com/wikia/webdriver/testcases/chattests/ChatTestsStaff.java [hunk 14]
```diff
@@ -11,8 +11,9 @@
...
+  private final static String MESSAGE_ON_PRIVATE_CHAT = "Test message on private chat";
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/chattests/ChatTestsStaff.java]: MINOR Reorder the modifiers to comply with the Java Language Specification. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AModifiersOrderCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 7 issues:
* MINOR 7 minor

Watch the comments in this conversation to review them.

**Comment by ludwikkazmierczak** [comment]
LGTM

**Pull request merged**